api/util.go

```diff
@@ -65,3 +66,12 @@ func MustLibp2pMultiaddrJoin(addr ma.Multiaddr, p peer.ID) ma.Multiaddr {
 	}
 	return addr.Encapsulate(pidAddr)
 }
+
+// TrimSlacesAndSpaces trims any leading and trailing spaces and backslaces to give the path
+// that can be added to a url string
+func TrimSlacesAndSpaces(p string) string {
```

Comment:
I'd like that this is not used anywhere and we do not sanitize bad paths, but error on wrong ones when we process them on the API server (or Cluster).